layer/clients/label.py

```diff
@@ -45,6 +49,18 @@ def add_labels_to(
         )
         self._service.AddLabelsToEntity(request=request)
 
+    def add_labels_to_run(
+        self,
+        run_id: UUID,
+        label_names: Set[str],
+    ) -> None:
+        request = AddLabelsToRunRequest(
+            run_id=RunId(value=str(run_id)),
```

Comment:
You can use the proto mapper here: https://github.com/layerai/sdk/blob/main/layer/clients/protomappers/runs.py#L11